packages/language-server/test/plugins/typescript/features/DiagnosticsProvider.test.ts

```diff
@@ -43,7 +43,7 @@ describe('DiagnosticsProvider', function () {
         const newFilePath = normalizePath(path.join(testDir, 'doesntexistyet.js')) || '';
         writeFileSync(newFilePath, 'export default function foo() {}');
         assert.ok(existsSync(newFilePath));
-        await lsAndTsDocResolver.getSnapshot(newFilePath);
+        await lsAndTsDocResolver.invalidateModuleCache(newFilePath);
```

Comment:
This test is the one that failed because `getSnapshot` no longer adds new root files, and it's the reason why I use the internal API `hasInvalidatedResolutions`.